changelog/unreleased/breaking-changes/1657.md

```diff
@@ -0,0 +1,4 @@
+The `configure` script was removed. This was a custom script that
+mimicked the functionality of an autotools-based `configure` script
+by writing directly to the cmake cache. Instead, users now must use
+the `cmake` and/or `ccmake` binaries directly to configure VAST.
```

Comment:
We have a ton of variables in our CMake that are not `mark_as_advanced` for `ccmake`. Can you create a follow-up story for cleaning that up?